scalding-core/src/main/scala/com/twitter/scalding/TypedDelimited.scala

```diff
@@ -47,15 +48,15 @@ trait TypedSeperatedFile extends Serializable {
 /**
  * Typed tab separated values file
  */
-object TypedTsv extends TypedSeperatedFile {
-  val separator = "\t"
+object TypedTsv {
+  def apply[T: TypeDescriptor](path: String*): TypedTextDelimited[T] = TypedText.tsv[T](path: _*)
 }
 
 /**
  * Typed comma separated values file
  */
-object TypedCsv extends TypedSeperatedFile {
```

Comment:
top lel, every instance of `TypedSeperatedFile` in our code base spells the word "separated" wrong!